.github/workflows/deploy.yml

```diff
@@ -446,24 +469,11 @@ jobs:
     if: ${{ github.ref == 'refs/heads/staging' && fromJSON(needs.checkDeploymentSuccess.outputs.IS_AT_LEAST_ONE_PLATFORM_DEPLOYED) }}
     needs: [checkDeploymentSuccess]
     steps:
-      # The command "gh release create" also creates and pushes a tag for the version if one does not already exist,
-      # So we need to checkout the staging branch and create that tag from the staging branch.
       - name: Checkout staging branch
         uses: actions/checkout@v4
-        with:
-          ref: staging
-          token: ${{ secrets.OS_BOTIFY_TOKEN }}
-
-      - name: Setup git for OSBotify
-        uses: ./.github/actions/composite/setupGitForOSBotifyApp
-        id: setupGitForOSBotify
-        with:
-          GPG_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
-          OS_BOTIFY_APP_ID: ${{ secrets.OS_BOTIFY_APP_ID }}
-          OS_BOTIFY_PRIVATE_KEY: ${{ secrets.OS_BOTIFY_PRIVATE_KEY }}
 
       - name: Get current app version
-        run: echo "STAGING_VERSION=$(npm run print-version --silent)" >> "$GITHUB_ENV"
+        run: echo "STAGING_VERSION=$(jq -r .version < package.json)" >> "$GITHUB_ENV"
```

Comment:
What's the reason for this change? Is it cause it's just faster reading from `package.json` with `jq`?

Reply:
commented above, it really doesn't matter. Maybe faster because it doesn't have to go through npm 🤷🏼

We use both styles around the repo

Reply:
Ah you already commented on that I see :D